x-pack/test_serverless/api_integration/test_suites/observability/burn_rate_rule/burn_rate_rule.ts

```diff
@@ -200,6 +279,18 @@ export default function ({ getService }: FtrProviderContext) {
         expect(executionStatus).to.be('active');
       });
 
+      it('should set correct information in the alert document', async () => {
```

Comment:
💯 .

Might be good to include api integration tests for the instanceId logic. I'd accept either api integration tests or unit tests.